.github/workflows/build.yml

```diff
@@ -103,8 +115,14 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup
+        id: setup
         if : ( steps.skip-check.outputs.skiptest != 'true' )
         run: |
+          if [ "${{ matrix.runs-on }}" = "ubuntu-24.04-arm" ]; then
+              echo "cpu=ARM64" >> $GITHUB_OUTPUT
+          else
+              echo "cpu=X86_64" >> $GITHUB_OUTPUT
+          fi
```

Comment:
what is the meaning of this change. It is not described in the commit message and from what I see, the `setup` is not used anywhere else later. Is it just something for debugging?

Reply:
it is used to name the artifacts in case of test failure, otherwise the x86_64 ones would have the same name as the arm64 ones and would overwrite each other